assemblers/org.osbuild.qemu

```diff
@@ -674,14 +675,31 @@ def main(tree, output_dir, options, loop_client):
             "vpc": ["-o", "subformat=fixed,force_size"],
             "vhdx": []
         }
-        subprocess.run([
-            "qemu-img",
-            "convert",
-            "-O", fmt,
-            *extra_args[fmt],
-            image,
-            f"{output_dir}/{filename}"
-        ], check=True)
+        # A bug exists in qemu that causes the conversion to raw to fail
+        # on aarch64 systems with a LOT of CPUs. A workaround is to use
+        # a single coroutine to do the conversion. It doesn't slow down
+        # the conversion by much, but it hangs about half the time without
+        # the limit set. 😢
+        # Bug: https://bugs.launchpad.net/qemu/+bug/1805256
+        if platform.machine() == 'aarch64' and fmt.startswith('raw'):
```

Comment:
Hm, this branch should not be taken. We don't convert `raw` or `raw.xz` image types using `qemu-img`. Are there other image types that start with `raw` where this applies?

Reply:
Ah, then this might be a non-issue in osbuild. I didn't run any thorough testing on this.

Reply:
Nice. The image file that osbuild produces is `raw`, so when that's the output format it is just a noop. I guess in image info it is the other way around (converts from whatever it is to `raw`).